flowetl/etl/etl/etl_utils.py

```diff
@@ -107,42 +111,25 @@ def construct_etl_dag(
     """
 
     with DAG(
-        dag_id=f"etl_{cdr_type}", schedule_interval=None, default_args=default_args
+        dag_id=f"etl_{cdr_type}",
+        schedule_interval=None,
+        default_args=default_args,
+        template_searchpath=config_path,
     ) as dag:
 
-        init = PythonOperator(
-            task_id="init", python_callable=init, provide_context=True
+        init = init(task_id="init")
+        extract = extract(task_id="extract", sql=f"/etl/{cdr_type}/extract.sql")
```

Comment:
Is the `sql` path relative to the `template_searchpath`/`config_path` defined above? I.e. does this path refer to `/mounts/config/etl/{cdr_type}/extract.sql`?

If so, is it possible to remove the leading slash to avoid it being interpreted as an absolute path (or does Airflow require the leading slash)?
```suggestion
        extract = extract(task_id="extract", sql=f"etl/{cdr_type}/extract.sql")
```

Reply:
Yup, it's relative to the `template_searchpath` from above, added a short comment to reflect this. The leading slash wasn't needed indeed, good point - removed from the paths.